metadata-models/src/main/pegasus/com/linkedin/feedback/SurveyResponseInfo.pdl

```diff
@@ -0,0 +1,31 @@
+namespace com.linkedin.feedback
+import com.linkedin.common.Time
+
+/*
+ * Response object for each response to a question in the user experience survey
+ */
+@Aspect = {
+  "name": "surveyResponseInfo"
+}
+
+record SurveyResponseInfo {
+    /*
+     * date when SurveyResponseInfo was created
+     */
+    createdAt: optional Time,
+
+    /*
+     * users response to a question
+     */
+    responseText: string,
+
+    /*
+     * question asked to user
+     */
+    questionText: string,
+
+    /*
+     * user's id
+     */
+    user: string,
```

Comment:
to indicate that an id is specifically a Urn, we usually use the `Urn` type. It's just an alias for string but is more explicit.